definitions/json/CaseField.json

```diff
@@ -3094,7 +3094,7 @@
   },
   {
     "CaseTypeID": "ET_EnglandWales",
-    "ID": "selectReferralToReply",
+    "ID": "selectReferral",
```

Comment:
As I said for the data model, shouldn't be changing IDs but it's 'ok' given this isn't live yet